.gitignore

```diff
@@ -10,3 +10,4 @@ terraform.tfstate.backup
 lib/
 *.lic
 /resources/
+coverage.txt
```

Comment:
We may also need to add `.scannerwork` as well. That directory appears in my local Artifactory provider directory.